INSTALL.md

```diff
@@ -56,3 +56,19 @@ dokku$ sudo mkdir -p /var/log/journal
 dokku$ dokku enter job-server
 container$ python manage.py sendtestemail me@myemail.org
 ```
+
+## Rotating the Django secret key
+
+If this is a routine rotation of the secret key,
+as opposed to a compromised secret key,
+you can temporarily leave the secret key to be replaced active:
+
+```bash
+# Store the existing SECRET_KEY as OLD_SECRET_KEY.
+dokku config:get job-server SECRET_KEY
+dokku config:set job-server OLD_SECRET_KEY='xxx'
```

Comment:
I bet you could even do something like `dokku config:set job-server OLD_SECRET_KEY=$(dokku config:get job-server SECRET_KEY)` and then there's no risk of secrets ending up in bash history.